packages/KIcon/KIcon.vue

```diff
@@ -133,6 +133,15 @@ export default {
       return this.svg ? (this.viewBox || (this.svg && this.svg.getAttribute('viewBox')) || DEFAULTS.viewBox) : DEFAULTS.viewBox
     }
   },
+  watch: {
+    // Ensure icon color is updated when color prop value changes
+    color: {
+      handler () {
+        this.svg = this.$refs.svgWrapper ? this.$refs.svgWrapper.querySelector('svg:not(.slot-content)') : null
```

Comment:
I’m unsure about setting `this.svg` to `null` here. Unsure what else this could cause

Reply:
I pulled that from https://github.com/Kong/kongponents/blob/main/packages/KIcon/KIcon.vue#L140, should we be doing this differently in both places, or just on this update?

Reply:
@adamdehaven bumping my question above 👆

Reply:
Oops, I missed this.

So we setup `this.svg` to the element or`null` in the `mounted()` function.. why does it need to be set in the watcher as well? Can't the function you're calling access `this.svg` already if it is set?

Reply:
@dustinryerson ^

Reply:
@adamdehaven nope you are totally right, I don't need that line. Thanks for catching that 👍